src/test/regress/expected/multi_alter_table_add_constraints.out

```diff
@@ -0,0 +1,131 @@
+--
```

Comment:
Let's also add test cases when we fail to create constraints because of;
* missing constraint name
* ADD CONSTRAINT is not only subcommand
* Constraint not on distribution column
* other...

Also let's add test for reference/append distributed tables too